resources/translations/en.edn

```diff
@@ -170,15 +170,15 @@
          :apply "Apply"
          :apply-for-bundle "Apply for %1 resources as a bundle "
          :header "%1 resources in the cart"
-         :intro [:div.intro [:p "Add resources to the cart to apply for them. If several resources can be applied to with the same application, they will be automatically grouped into a bundle."]]
+         :intro [:div.intro [:p "Apply for access by adding resources to the cart. When applying for multiple rights at the same time, resources using the same application process are bundled together as one application."]]
```

Comment:
This is the only place in the translations in which the word "right" is used to refer to the access rights. Using "access right" instead ("When applying for multiple access rights [...]") would add some context.

Reply:
"Apply for access rights by adding…" Term access by it self refers to physical access. "When applying for multiple resources…". An also, ""…using the same application process are bundled together as a single application" might be even more clear?

Reply:
"Apply for access by adding resources to the cart. When applying for access to multiple resources at the same time, resources using the same application are bundled together."

Reply:
Added "[Apply for access] rights" to the first sentence, as Jaakko proposed.